k8s/upgrades/0.8.0-0.8.1/cstor-target-patch.tpl.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I am not too sure of the file naming convention.
Should we name the file based on what is patched?
Should there be multiple files where each file does one patch?
For Example: a cstor-target-patch-sc-in-annotations.tpl.json

cc @kmova